index.html

```diff
@@ -2624,12 +2618,17 @@ <h2>WoT Runtime</h2>
             <p>
                 Technically, the <a>Thing</a> abstraction and its <a>Interaction Model</a> is implemented in a runtime system.
                 This <a>WoT Runtime</a> maintains the execution environment for the behavior implementation and is able to expose and/or consume <a>Things</a>,
-                and hence must be able to process, serialize, serve, and fetch <a>WoT Thing Descriptions</a>.
+                and hence must be able to fetch, process, serialize, and serve <a>WoT Thing Descriptions</a>.
             </p>
             <p>
-                The <a>WoT Runtime</a> must provide an API for the behavior implementation that follows the <a>Thing</a> abstraction.
-                The optional <a>WoT Scripting API</a> building block shown in <a href="#architecture-implementation"></a> defines such an application-facing interface.
+                The <a>WoT Runtime</a> must provide an API for the behavior implementation.
```

Comment:
Is this normative? If yes, you must define what kind of API is normative there. 
But this should not be normative, since a WoT Runtime may contain its applications (Things, behaviors) and manufacturers should be free to do so, without the requirement to expose non-specified APIs.

Reply:
No RFC 2119 langauge here. Still, there is naturally always an API, otherwise no application logic can be specified. This can and often is an API at compile time when no scripting is involved.

Reply:
Then you must specify what kind of API do you mean, at least on functional requirements terms. Saying "there is naturally always an API" doesn't mean anything, since between every code module there is an API, but that is no good reason to talk about an API in a normative document without specifics.